SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,87 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition []'),'2;1');
+FILE_NAME('SectionedSurface-Aplitop-1.ifc','2021-06-03T13:37:56',(),(),'IfcOpenShell 0.5.0-dev','IfcOpenShell 0.5.0-dev','');
+FILE_SCHEMA(('IFC4X3_RC3'));
+ENDSEC;
+DATA;
+#1=IFCACTORROLE(.USERDEFINED.,'',$);
+#2=IFCPOSTALADDRESS(.HOME.,$,$,$,(''),$,$,$,$,$);
+#3=IFCPERSON('Javier','','',$,$,$,(#1),(#2));
+#4=IFCPOSTALADDRESS(.OFFICE.,$,$,$,(''),$,$,$,$,$);
+#5=IFCORGANIZATION('','Aplitop','',$,(#4));
+#6=IFCPOSTALADDRESS(.OFFICE.,$,$,'',(''),$,'','','','');
+#7=IFCAPPLICATION(#5,'8.5','TCP-Digital Terrain Model','TcpMDT');
+#8=IFCPERSONANDORGANIZATION(#3,#5,$);
+#9=IFCOWNERHISTORY(#8,#7,$,$,$,$,$,0);
+#10=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#11=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#12=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#13=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#14=IFCUNITASSIGNMENT((#10,#11,#12,#13));
+#15=IFCDIRECTION((1.,0.,0.));
+#16=IFCDIRECTION((0.,0.,1.));
+#17=IFCCARTESIANPOINT((0.,0.,0.));
+#18=IFCAXIS2PLACEMENT3D(#17,#16,#15);
+#19=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#18,$);
+#20=IFCDIRECTION((1.,0.,0.));
+#21=IFCDIRECTION((0.,0.,1.));
+#22=IFCCARTESIANPOINT((0.,0.,0.));
+#23=IFCAXIS2PLACEMENT3D(#22,#21,#20);
+#24=IFCLOCALPLACEMENT($,#23);
+#25=IFCSITE('2dmSYY_dfEUvS$EqHsga9H',#9,'Site',$,$,#24,$,$,.ELEMENT.,$,$,$,$,#2);
+#26=IFCPROJECT('3tkRESp0n1qQfvDcC911af',#9,'Project','',$,'',$,$,#14);
+#27=IFCRELAGGREGATES('1J78ZkUl142R8c9zRv8Nuh',#9,$,$,#26,(#25));
+#28=IFCROAD('1b7reoKwTFJxr7W5culxb$',$,'Road',$,'IfcRoad',$,$,$,.COMPLEX.,$);
+#29=IFCRELAGGREGATES('1nbAt7eYv4swrQ7kMUYrD3',#9,$,$,#25,(#28));
+#30=IFCALIGNMENT('3NYWW0yfD9BwdMM3yK9zLG',$,'Eje1',$,$,#24,$,$);
+#31=IFCALIGNMENTHORIZONTAL('1Lb11OMQ1AthVTi6V7jHiU',$,'Eje1',$,$,#24,$,0.);
+#32=IFCRELNESTS('3Wkl6gqJj9$RKLoFB4xbkt',#9,$,$,#30,(#31,#51));
+#33=IFCALIGNMENTSEGMENT('0sVreI8Kn6tuw4UYDmeHtA',$,$,$,$,#24,$,#35);
+#34=IFCCARTESIANPOINT((286.47559897,239.79766567));
+#35=IFCALIGNMENTHORIZONTALSEGMENT($,$,#34,1.50346536986339,0.,0.,107.15812752,$,.LINE.);
+#36=IFCDIRECTION((0.999655739909883,0.0262374096515884));
+#37=IFCAXIS2PLACEMENT2D(#34,#36);
+#40=IFCCARTESIANPOINT((0.,0.));
+#41=IFCDIRECTION((1.,0.));
+#42=IFCVECTOR(#41,1.);
+#43=IFCLINE(#40,#42);
+#44=IFCCURVESEGMENT(.CONTINUOUS.,#37,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.15812752),#43);
+#45=IFCRELNESTS('3U4tk8m5LAnhDZNYLR6Op1',#9,$,$,#31,(#33));
+#46=IFCCOMPOSITECURVE((#44),.F.);
+#47=IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#19,$,.GRAPH_VIEW.,$);
+#48=IFCSHAPEREPRESENTATION(#47,'Axis','Curve2D',(#46));
+#49=IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
```

Comment:
This line should be referenced somewhere.

Reply:
In updated file, #49 is representation of IFCHORIZONTALALINGMENT